extension/src/experiments/columns/extract.ts

```diff
@@ -21,28 +25,40 @@ const extractMetricsOrParams = (
   return acc
 }
 
-const extractDeps = (columns?: Deps): DepColumns | undefined => {
+const extractDeps = (
+  columns?: Deps,
+  branch?: Experiment
+): DepColumns | undefined => {
   if (!columns) {
     return
   }
 
   const acc: DepColumns = {}
 
   for (const [path, { hash }] of Object.entries(columns)) {
-    acc[path] = shortenForLabel(hash)
+    const value = shortenForLabel(hash)
+    acc[path] = value
+
+    if (value && branch && branch?.deps?.[path] !== value) {
```

Comment:
[Q] Is this definitely accessing the right property in the branch?

Reply:
I can see from 
```
export interface DepColumns {
  [path: string]: string | ValueWithChanges
}
```
that you've got mixed types. It would be a good idea to standardise the type now as we are moving towards Studio style tooltips for deps:

![image](https://user-images.githubusercontent.com/37993418/179134442-fbb687de-5b1b-4863-bfcb-a6e84f214311.png)